### ISSUE TITLE
Changing the batch size

### ISSUE THREAD

**Issue by GoodDeeds** [GriffinLiang/vrd-dsr#14]
Is it possible to change the batch size while training? I see that there is a [`_batch_size`](https://github.com/GriffinLiang/vrd-dsr/blob/5a5ae98cb4f3558c0b42df30e699e1793476dd0e/lib/data_layers/vrd_data_layer.py#L42) variable in the `VRDDataLayer` class, but the only place it is used seems to be [here](https://github.com/GriffinLiang/vrd-dsr/blob/5a5ae98cb4f3558c0b42df30e699e1793476dd0e/lib/model.py#L20) for setting the number of epochs. While actually doing the [forward pass](https://github.com/GriffinLiang/vrd-dsr/blob/master/lib/data_layers/vrd_data_layer.py#L58), it seems that only one image is used.

Is this the case? Will training code have to be modified to allow a higher batch size?

Thank you.

**Comment by GriffinLiang** [comment]
I think it's possible to use more data in one batch. You need to modify the codes related to roi pooling and the ranking loss.
